WooCommerce/Classes/Authentication/WPComLogin/WPComEmailLoginViewModel.swift

```diff
@@ -168,5 +173,10 @@ extension WPComEmailLoginViewModel {
             "share details",
             comment: "The action to be agreed upon when tapping the Connect Jetpack button on the Wrong Account screen."
         )
+        static let unknownUsername = NSLocalizedString(
+            "wpComEmailLoginViewModel.unknownUsername",
+            value: "Hmm, we can\'t find a WordPress.com account connected to this username. You can enter an email to create a new account.",
```

Comment:
np: "hmm" is a [filler word](https://www.grammarly.com/blog/grammar/how-we-use-filler-words/). Ideally it can be removed to make the UI message more concise https://www.interaction-design.org/literature/article/8-essential-writing-tips-ux-ui-portfolio#why_should_you_limit_filler_words?-11

Reply:
Additionally, I think the button on the error modal should say "OK" instead of "Cancel". What do you think?

Reply:
> np: "hmm" is a [filler word](https://www.grammarly.com/blog/grammar/how-we-use-filler-words/)

The word `hmm` is taken from the Android part, we had it in the old error message, and @JorgeMucientes just adjusted it for this case. I'm fine with removing it, but I think keeping both platforms aligned is more valuable. @JorgeMucientes WDYT about this?

> Additionally, I think the button on the error modal should say "OK" instead of "Cancel". What do you think?

Do you think this should apply to all error messages or just this one? Personally I think OK is a better choice for the other errors as well, but I didn't want to change something that was already there.

Reply:
I'm fine with removing the `hmm` wording. Just to be clear the final text would be `We can't find a WordPress.com account connected to this username. You can enter an email to create a new account.`
Correct? 
I'll add that change alongside the tracking change we agreed on.

Reply:
OK, thanks @JorgeMucientes, for iOS, I updated the message on [059d0f0](https://github.com/woocommerce/woocommerce-ios/pull/14454/commits/059d0f03376e8d53ec36252d5ec35b3482f0e9b4)

Reply:
For the cancel button @hafizrahman, I'm merging this as it's right now to not miss the code freeze, we can update the text if needed later.